src/bin/pg_autoctl/cli_drop_node.c

```diff
@@ -578,9 +578,44 @@ cli_drop_local_node(KeeperConfig *config, bool dropAndDestroy)
 	 */
 	pid_t pid = 0;
 
-	if (!wait_for_pid_to_exit(config->pathnames.pid, 30, &pid))
+	/*
+	 * Before continuing we need to make sure that a currently running service
+	 * has stopped.
+	 *
+	 * If --force isn't used then a running pg_autoctl process will detect that
+	 * it is dropped and clean itself up nicely and finally it will exit. We
+	 * give the process 30 seconds to exit by itself.
+	 *
+	 * If --force is used, we skip the transition to "dropped". So a currently
+	 * running process won't realise it's dropped, so it will not exit by
+	 * itself. So there's no point in waiting for 30 seconds.
+	 */
+	bool stopped;
+	if (!dropForce)
```

Comment:
Nitpick: if we're going to have code paths for the both true and false values, can we do them in the easy to read order, `if (dropForce) { ... } else { ... }`?